signer/threshold_ed25519_localHSMsign_signer.go

```diff
@@ -0,0 +1,59 @@
+package signer
+
+import (
+	"errors"
+)
+
+// Inistiates the LocalHSMsignThresholdEd25519SignatureConfig
+func NewLocalHSMSignThresholdEd25519SignatureConfig(
+	cfg LocalCosignerConfig) LocalHSMsignThresholdEd25519SignatureConfig {
+
+	panic("Not Implemented")
+}
+
+// Placeholder for Local LocalHSMsignThresholdEd25519SignatureConfig
+type LocalHSMsignThresholdEd25519SignatureConfig struct {
+}
+
+func (cfg *LocalHSMsignThresholdEd25519SignatureConfig) NewThresholdEd25519Signature() ThresholdEd25519Signature {
+	panic("Not Implemented")
+
+	// return localsigner
+}
+
+type LocalHSMsignThresholdEd25519Signature struct {
+	// TODO: Implement HSM Signer
+}
+
+func (localsigner *LocalHSMsignThresholdEd25519Signature) DealShares(
+	req CosignerGetEphemeralSecretPartRequest) (HrsMetadata, error) {
+
+	// panic("Not Implemented")
+	return HrsMetadata{}, errors.New("method dealShares not implemented")
```

Comment:
We can be consistent on all of these with the `panic("Not Implemented")`

Reply:
"re-instated" panic("Not Implemented") and commented the return statements out.